### PR TITLE
Set most accurate permission type for error logging

### LINKED ISSUE #162: Clean up log messages for 1.0
The log contains a lot of noise and information (such as URLs) that probably shouldn't be in the log. Clean up logs and ensure log levels are correct (i.e. Log.e is actually an error).

### PR DESCRIPTION
**Pull request by keianhzo**
Fixes #162 

I've tried to set the most accurate log level. Regarding removing log info, @bluemarvin  why instead of removing debug info we just strip all debug messages for release?

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/ui/BrowserWidget.java [hunk 1]
```diff
@@ -122,7 +122,7 @@ public WidgetPlacement getPlacement() {
     @Override
     public void handleTouchEvent(MotionEvent aEvent) {
         if (aEvent.getActionMasked() == MotionEvent.ACTION_DOWN) {
-            Log.e(LOGTAG, "************** REQUESTING FOCUS");
+            Log.d(LOGTAG, "************** REQUESTING FOCUS");
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/BrowserWidget.java]: I would remove this log as it was just there for debugging during development. I'm not sure it adds any value now.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/KeyboardWidget.java [hunk 2]
```diff
@@ -83,7 +83,7 @@ private void initialize(Context aContext) {
         mKeyboardview.setOnKeyListener(new View.OnKeyListener() {
             @Override
             public boolean onKey(View view, int i, KeyEvent keyEvent) {
-                Log.e("VRB", "OnKeyListener: " + i + " " + keyEvent.toString());
+                Log.d("VRB", "OnKeyListener: " + i + " " + keyEvent.toString());
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/KeyboardWidget.java]: We probably shouldn't be logging keystrokes.

File: app/src/main/cpp/ControllerContainer.cpp [hunk 3]
```diff
@@ -175,7 +175,7 @@ ControllerContainer::CreateController(const int32_t aControllerIndex, const int3
         m.root->ToggleChild(*controller.transform, false);
       }
     } else {
-      VRB_LOG("FAILED TO ADD MODEL");
+      VRB_ERROR("FAILED TO ADD CONTROLLER MODEL");
```
Comment (bluemarvin) [inline app/src/main/cpp/ControllerContainer.cpp]: This probably shouldn't be in all caps, we don't need to shout about it 😄

File: app/src/main/cpp/FadeBlitter.cpp [hunk 4]
```diff
@@ -78,7 +78,7 @@ FadeBlitter::Create(vrb::CreationContextPtr& aContext) {
 void
 FadeBlitter::Draw() {
   if (!m.program) {
-    VRB_LOG("FadeBlitter::Draw FAILED!");
+    VRB_ERROR("FadeBlitter::Draw FAILED!");
```
Comment (bluemarvin) [inline app/src/main/cpp/FadeBlitter.cpp]: This could be maybe more detailed stating why it failed?

**Comment by keianhzo** [comment]
Fixes those. I'm going to add a task to setup a Proguard to strip debug and verbose logs for release APK as I think it's useful to have things like URLs logged during development. It require some work as we need to also include rules for third party libs including Gecko.

File: app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java [hunk 5]
```diff
@@ -220,9 +219,9 @@ void loadFromIntent(final Intent intent) {
             int id = SessionStore.get().createSession();
             SessionStore.get().setCurrentSession(id);
             SessionStore.get().loadUri(url);
-            Log.e(LOGTAG, "Load creating session and loading URI:" + url);
+            Log.d(LOGTAG, "Load creating session and loading URI:" + url);
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java]: This message should probably be "Creating session and loading URI:"

File: app/src/common/shared/org/mozilla/vrbrowser/ui/UIWidget.java [hunk 6]
```diff
@@ -59,7 +59,7 @@ private void initialize() {
     @Override
     public void setSurfaceTexture(SurfaceTexture aTexture, final int aWidth, final int aHeight) {
         if (mTexture!= null && (mTexture.equals(aTexture))) {
-            Log.e(LOGTAG, "TEXTURE ALREADY SET");
+            Log.e(LOGTAG, "Texture already set");
```
Comment (bluemarvin) [inline app/src/common/shared/org/mozilla/vrbrowser/ui/UIWidget.java]: This isn't an error so probably should just change it to debug log.

File: app/src/main/cpp/BrowserWorld.cpp [hunk 7]
```diff
@@ -506,7 +506,7 @@ BrowserWorld::Draw() {
   if (!m.glInitialized) {
     m.glInitialized = m.context->InitializeGL();
     if (!m.glInitialized) {
-      VRB_LOG("FAILED to initialize GL");
+      VRB_ERROR("FAILED to initialize GL");
```
Comment (bluemarvin) [inline app/src/main/cpp/BrowserWorld.cpp]: Due to the Android lifecycle, this isn't really an error. Probably okay to leave as a VRB_LOG.

File: app/src/main/cpp/GeckoSurfaceTexture.cpp [hunk 8]
```diff
@@ -207,7 +207,7 @@ GeckoSurfaceTexture::DecrementUse() {
 GeckoSurfaceTexture::GeckoSurfaceTexture(State& aState) : m(aState) {}
 GeckoSurfaceTexture::~GeckoSurfaceTexture() {
   if (m.surface) {
-    VRB_LOG("**** DESTROY GeckoSurfaceTexture");
+    VRB_LOG("**** Destroy GeckoSurfaceTexture");
```
Comment (bluemarvin) [inline app/src/main/cpp/GeckoSurfaceTexture.cpp]: Remove the leading ****

**Pull request merged**